tasks/main.yml

```diff
@@ -23,6 +23,7 @@
     name: "{{ item }}"
     state: present
   with_items: "{{ ice_devel_packages }}"
+  when: 'ice_install_devel | default(True)'
```

Comment:
General question: is there a trade-off to declaring a default value for `ice_install_devel` under `defaults/main.yml` and reducing this conditional to `when: ice_install_devel` compared to this approach?